service/src/main/java/greencity/service/ubs/SuperAdminServiceImpl.java

```diff
@@ -151,96 +180,79 @@ public GetTariffServiceDto editTariffService(EditTariffServiceDto dto, Integer i
     }
 
     @Override
-    public CreateServiceDto addService(CreateServiceDto dto, String uuid) {
-        User user = userRepository.findByUuid(uuid);
-        Service service = createServiceWithTranslation(dto, user);
-        service.setFullPrice(getFullPrice(dto.getPrice(), dto.getCommission()));
-        serviceRepository.save(service);
-        serviceTranslationRepository.saveAll(service.getServiceTranslations());
-        return modelMapper.map(service, CreateServiceDto.class);
+    public ServiceDto addService(CreateServiceDto dto, String uuid) {
+        Service service = serviceRepository.save(createService(dto, uuid));
+        return modelMapper.map(service, ServiceDto.class);
     }
 
-    private Service createServiceWithTranslation(CreateServiceDto dto, User user) {
-        Long id = dto.getCourierId();
-        Courier courier = courierRepository.findById(id).orElseThrow(
-            () -> new NotFoundException(ErrorMessage.COURIER_IS_NOT_FOUND_BY_ID + id));
-
-        Service service = Service.builder()
-            .basePrice(dto.getPrice())
-            .commission(dto.getCommission())
-            .fullPrice(dto.getPrice() + dto.getCommission())
-            .capacity(dto.getCapacity())
-            .createdAt(LocalDate.now())
-            .courier(courier)
-            .createdBy(user.getRecipientName() + " " + user.getRecipientSurname())
-            .serviceTranslations(dto.getServiceTranslationDtoList()
-                .stream().map(serviceTranslationDto -> ServiceTranslation.builder()
-                    .description(serviceTranslationDto.getDescription())
-                    .descriptionEng(serviceTranslationDto.getDescriptionEng())
-                    .name(serviceTranslationDto.getName())
-                    .nameEng(serviceTranslationDto.getNameEng())
-                    .build())
-                .collect(Collectors.toList()))
-            .build();
-        service.getServiceTranslations().forEach(serviceTranslation -> serviceTranslation.setService(service));
-        return service;
+    private Service createService(CreateServiceDto dto, String uuid) {
```

Comment:
Please rename `uuid` in all flow
I thought that it is service's `uuid`
`uuid -> employeeUuid`

Reply:
changed